pic32/variants/Uno32/Board_Defs.h

```diff
@@ -155,6 +155,9 @@
 #define	PIN_INT3	8
 #define	PIN_INT4	35
 
+#define NOT_AN_INTERRUPT -1
+#define digitalPinToInterrupt(p) ((p) == PIN_INT0 ? 0 : ((p) == PIN_INT1 ? 1 : ((p) == PIN_INT2 ? 2 : ((p) == PIN_INT3 ? 3 : ((p) == PIN_INT4 ? 4 : NOT_AN_INTERRUPT)))))
```

Comment:
Yes! it is required for MANY other libraries. It is expected to be available, even if the core does not use it. It needs to also be ported to all other platforms.